module/VuFind/src/VuFind/Role/PermissionProvider/PluginManager.php

```diff
@@ -46,6 +46,7 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
      * @var array
      */
     protected $aliases = [
+        'cookie' => Cookie::class,
```

Comment:
I also added this missing alias for you. :-)

Reply:
thanks @demiankatz